### ISSUE TITLE
Slog mirror update?

### ISSUE THREAD

**Issue by gamejunk58** [openzfs/openzfs-docs#281]
I have checked the docs about my doubts but I am still unsure. 

There is a lot of information online stating that slog devices should have power loss protection and be mirrored. 
(ignoring TBW and speed/latency for a moment)

https://www.reddit.com/r/freenas/comments/cd3t4m/is_a_slog_zil_cache_nvme_ssd_worth_it/
However the top commenter here says the mirroring part is outdated information and there is 0 risk in having a slog device completely fail out of nowhere. Is this true? Or did he get it mixed up with L2ARC?

If this is 100% true, then why do you still care about power loss protection in the docs if it doesn't matter if the whole device gets broken? I can't make sense of this. 

I need to know this before I install a slog device. And I thought that if there is so much outdated false info online as this guy says shouldn't it be mentioned in the docs?

**Comment by gmelikov** [comment]
There is one case when slog mirror will help - after power loss (i.e. pool didn't write data from slog into main vdevs yet) your slog disk may be dead, so you'll have an up to txg_timeout sync writes lost. The main case to have slog is to ensure that already synced data (for ex. money transfer transaction) is safe on disks.

tl;dr: faulted slog on powerloss will eat up to txg_timeout period of synced data.
